### PR TITLE
Scala 2.12.0 support

### PR DESCRIPTION
**Pull request by golem131**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 0.13.11
+sbt.version = 0.13.12
```
Comment (hseeberger) [inline project/build.properties]: Please go for 0.13.13.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import sbt._
 
 object Version {
-  final val Akka       = "2.4.11"
+  final val Akka       = "2.4.12"
   final val Log4j      = "2.6.2"
   final val Scala      = "2.11.8"
```
Comment (hseeberger) [inline project/Dependencies.scala]: Please set the "main" Scala version to 2.12.0.

File: project/Build.scala [hunk 3]
```diff
@@ -33,7 +33,7 @@ object Build extends AutoPlugin {
                   </developer>
                 </developers>,
     scalaVersion := Version.Scala,
-    crossScalaVersions := Vector(scalaVersion.value),
+    crossScalaVersions := Vector(scalaVersion.value, "2.12.0"),
```
Comment (hseeberger) [inline project/Build.scala]: Please change this according to the change of the "main" Scala version.

File: .travis.yml [hunk 4]
```diff
@@ -1,4 +1,10 @@
 language: scala
+
```
Comment (hseeberger) [inline .travis.yml]: This is not necessary because of `sbt +test`.

File: .travis.yml [hunk 5]
```diff
@@ -1,4 +1,10 @@
...
+scala:
+  - 2.12.0
+  - 2.11.8
+
 jdk:
-- oraclejdk8
-script: sbt +test
+  - oraclejdk8
+
```
Comment (hseeberger) [inline .travis.yml]: Please revert the whole file.

**Comment by golem131** [comment]
All fixed

File: .travis.yml [hunk 6]
```diff
@@ -1,4 +1,4 @@
 language: scala
 jdk:
 - oraclejdk8
-script: sbt +test
+script: sbt +test
```
Comment (guidomedina) [inline .travis.yml]: Revert this file, this change is not needed as nothing really has changed, you have to exclude this file from Git commit.

**Comment by golem131** [comment]
Now all ok?

HA-HA-HA, travis infrastructure error now :(

**Comment by guidomedina** [comment]
Looks good to me, the reason I commented on your commit was because I'm also interested on seeing akka-log4j available for Scala 2.12.0, I'm also following on the same task for akka-kryo :D

**Edit:** Using whip while screaming "faster faster !!!"

**Comment by hseeberger** [comment]
Thanks!

**Comment by hseeberger** [comment]
I have just released 1.1.6 for Scala 2.11 and 2.12.

**Pull request merged**